_tools/redirects/redirects.csv

```diff
@@ -1,8 +1,8 @@
 source,destination
 /classes/_classes.html,/classes/
 /community/tutorials/3d/mesh_generation_with_heightmap_and_shaders.html,/tutorials/3d/mesh_generation_with_heightmap_and_shaders.html
-/community/tutorials/gdnative/gdnative-c-example.html,/tutorials/plugins/gdnative/gdnative-c-example.html
-/community/tutorials/gdnative/index.html,/tutorials/plugins/gdnative/index.html
+/community/tutorials/gdextension/gdextension-c-example.html,/tutorials/plugins/gdextension/gdextension-c-example.html
+/community/tutorials/gdextension/index.html,/tutorials/plugins/gdextension/index.html
```

Comment:
This should not touch existing redirects, but add new ones at the bottom of the file.
Existing redirects still need to be setup a work for old branches.